File: leetcode2/1easy/이준열/3432.cpp
class Solution {
    public:
        int countPartitions(vector<int>& nums) {
            vector<int> prefixSum(nums.size());
            prefixSum[0] = nums[0];
            int output = 0;
    
            for (int i = 1; i < nums.size(); i++)
            {
                prefixSum[i] = nums[i] + prefixSum[i-1];
            }
    
            for (int j = 0; j < prefixSum.size() - 1; j++)
            {
                if ((2 * prefixSum[j] - prefixSum.back()) % 2 == 0)
                    output++;
            }
            return output;
        }
    };

File: leetcode2/1easy/유시연/3432.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def countPartitions(self, nums: List[int]) -> int:
        n = len(nums)
        count = 0
        for i in range(n-1):
            if sum(nums[:i+1]) % 2 == 0 and sum(nums[i+1:]) % 2 == 0:
                count += 1
            if sum(nums[:i+1]) % 2 != 0 and sum(nums[i+1:]) % 2 != 0:
                count += 1

        return count